### PR TITLE
[enh] Human readable date

### PR DESCRIPTION
**Pull request by kir0ul**
## The problem

The command `sudo yunohost app listlists` prints a not very understandable date, like the following:
```
yunohost:
  lastUpdate: 1538746930
```
See [issue 901](https://github.com/YunoHost/issues/issues/901) for more informations.

In general, date are not well managed, this PR try to harmonize the date management.

## Solution
Filename containing a date should be in UTC.
Files containing dates should use "aware" date format or naive UTC date to store the date.
If a timestamp is stored it should be convert with utcfromtimestamp .

CLI should display date for human and use the system timezone to display date (See https://github.com/YunoHost/moulinette/pull/184 )

Webadmin should display date in the broser locale timezone (See https://github.com/YunoHost/yunohost-admin/pull/216 )

Info: yunohost logs lines seems use asctime to store dates in system local timezone (without timezone info). I don't know if it's a good idea.

## Status
Ready
PR linked to 
https://github.com/YunoHost/yunohost-admin/pull/216
https://github.com/YunoHost/moulinette/pull/184

### CODE REVIEW

**Comment by zamentur** [comment]
We discussed this during the pycon fr. I think we should consider make this change by default (without option -H) and give a date (as moulinette and yunohost-admin contains a code to transmit date through API). 

We could add an option to get the timestamp but i think it's not necessary.

The goal of this change is to give the better UX has possible by default.

**Comment by alexAubin** [comment]
(Bump @zamentur  )

**Comment by kir0ul** [comment]
@zamentur @alexAubin Sorry for the late answer. I am really busy on other projects right now and I think I probably won't have time to work on this before a few weeks (maybe at Christmas). Sorry about that.

File: src/yunohost/app.py [hunk 1]
```diff
@@ -80,6 +82,11 @@ def app_listlists():
     # Get the list
     appslist_list = _read_appslist_list()
 
+    for app in appslist_list:
+        last_update = datetime.datetime.utcfromtimestamp(
+            appslist_list[app].get("lastUpdate"))
+        appslist_list[app]["lastUpdate"] = last_update.replace(tzinfo=pytz.utc)
```
[the remark this answers is not in the document]
Comment (zamentur) [reply src/yunohost/app.py]: without .replace the date is a naive date (without timezone info).

File: src/yunohost/monitor.py [hunk 2]
```diff
@@ -283,7 +283,7 @@ def monitor_system(units=None, human_readable=False):
         elif u == 'process':
             result[u] = json.loads(glances.getProcessCount())
         elif u == 'uptime':
-            result[u] = (str(datetime.now() - datetime.fromtimestamp(psutil.boot_time())).split('.')[0])
+            result[u] = (str(datetime.utcnow() - datetime.utcfromtimestamp(psutil.boot_time())).split('.')[0])
```
[the remark this answers is not in the document]
Comment (zamentur) [reply src/yunohost/monitor.py]: as it's a delta it's not needed

File: src/yunohost/app.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexAubin) [reply src/yunohost/app.py]: Couldn't we factorize this directly in moulinette's https://github.com/YunoHost/moulinette/pull/184 ?

**Comment by zamentur** [comment]
Ready and tested

**Pull request merged**